recipe/meta.yaml

```
@@ -49,18 +45,96 @@ requirements:
     - pytorch
     - pytorch =*=cuda*
     - setuptools
-  run:
-    - einops
-    - python
-    - pytorch =*=cuda*
 
-test:
-  imports:
-    - flash_attn
-  commands:
-    - pip check
-  requires:
-    - pip
+outputs:
+  - name: flash-attn
```

Think we just need to move those `ignore_run_exports_from` to this output as they were before

Also note that it appears we need `libcurand-dev` now. So think it would need to be in this list too

```suggestion
  - name: flash-attn
    build:
      ignore_run_exports_from:
        - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
        - libcublas-dev    # [(cuda_compiler_version or "").startswith("12")]
        - libcurand-dev    # [(cuda_compiler_version or "").startswith("12")]
        - libcusolver-dev  # [(cuda_compiler_version or "").startswith("12")]
        - libcusparse-dev  # [(cuda_compiler_version or "").startswith("12")]
```

---

The `libcurand-dev` is needed in [this header]( https://github.com/Dao-AILab/flash-attention/blob/478ee666cccbd1b8f63648633003059a8dc6827d/csrc/layer_norm/ln_fwd_kernels.cuh#L10 )

So dropping from `ignore_run_exports_from` above. Same with `libcublas-dev`